### DISCUSSION TITLE
Dialectics

### DISCUSSION THREAD

**Discussion by kennethsato** [SubtitleEdit/subtitleedit#6097] [Q&A] [answered]
Hello!

I'm using Substitle Edit to Convert (Brazillian portuguese) .srt to .scc
Whenever we have a special dialectic char on a caption, for example:

> 00:00:07,040 --> 00:00:09,291
> Sem tempo para ir
> ao salão de beleza?

We are loosing the previous char on scc. We get:

> 00:00:07,040 --> 00:00:09,291
> Sem tempo para ir
> ao saão de beleza?

Is there any option for fixing that?

**Comment by niksedk** [comment, marked as answer]
I cannot re-create this issue... what program are you reading the .scc file in?

**Reply by kennethsato** [reply]
I'm sorry @niksedk .
I think I mistakenly debugged my issue.
I am using Drastic Technologies's CCConvert to convert SubstitleEdit's SCC to a MCC file.
And then embedding it to my video also with CCConvert.

I thought SubtitleEdit was missing my dialectics.
Because I also tried to generate the scc by Drastic's CCConvert and it was OK with the chars (although on paint-on style)

But I have just read the SCC with this charset: http://www.theneitherworld.com/mcpoodle/SCC_TOOLS/DOCS/cc_charset.gif
And I've realized that SubstitleEdit's SCC is correct and CCConvert's that is duplicating chars (and then removing them on MCC's conversion).
